docs/credentials-with-issuer-dependent-terms.json

```diff
@@ -68,7 +68,7 @@
     "count": 0
   },
   {
-    "type": "OrganicCertificationCredential",
+    "type": "OrganicCertificateCredential",
```

Comment:
This seems out of place for this PR ... though it is certainly a change that should be made.

Reply:
Definitely - my bad

Reply:
This is due to a recent change, but does need to be merged in. Seems like it'd be fine to include this and merge.

Reply:
OK, thanks @rhofvendahl !